### PR TITLE
Use NLTK 2.0.6

### PR DESCRIPTION
**Pull request by singingwolfboy**
From edx's fork of NLTK

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -26,5 +26,7 @@ pytz==2015.2
 South==1.0.1
 voluptuous==0.8.5
 
+-e git+https://github.com/edx/nltk.git@2.0.6#egg=nltk
```
Comment (feanil) [inline requirements/base.txt]: Since this is also needed for AI grading, let's move this down below the `#AI grading` comment.

Comment (singingwolfboy) [inline requirements/base.txt]: Fixed.

File: requirements/base.txt [hunk 2]
```diff
@@ -27,4 +27,5 @@ South==1.0.1
 voluptuous==0.8.5
 
 # AI grading
-git+https://github.com/edx/ease.git@b67d2928a26fe497826b6ea359b9a3d0371548a7#egg=ease==0.1.3
+git+https://github.com/edx/ease.git@3d436b7763c7670f36e2e4630629002e8d01611e#egg=ease==0.1.3
```
Comment (feanil) [inline requirements/base.txt]: Let's use a release tag instead.

Comment (singingwolfboy) [inline requirements/base.txt]: Fixed.

File: requirements/base.txt [hunk 3]
```diff
@@ -27,4 +27,5 @@ South==1.0.1
 voluptuous==0.8.5
 
 # AI grading
-git+https://github.com/edx/ease.git@b67d2928a26fe497826b6ea359b9a3d0371548a7#egg=ease==0.1.3
+git+https://github.com/edx/ease.git@release-2015-07-09#egg=ease==0.1.3
+-e git+https://github.com/edx/nltk.git@2.0.6#egg=nltk
```
Comment (feanil) [inline requirements/base.txt]: Instead of `-e` let's the same format as ease above of actually pinning to the version.

Comment (singingwolfboy) [inline requirements/base.txt]: Fixed.

**Comment by feanil** [comment]
:+1:

**Pull request merged**